lib/registries/addon/overview/index/controller.ts

```diff
@@ -13,11 +13,20 @@ export default class Overview extends Controller {
     registration?: Registration;
 
     @computed('registration.{root,isRoot}')
-    get root() {
+    get rootTitle() {
```

Comment:
This code has been updated to process two separate computed properties. I have tested with one getter for the two separate values title and id used in the link and title values respectively but splitting them resulted in the desired code result. If there is a way to combine them to remove extra logic, it is something I am currently testing and would be greatly appreciated.

Reply:
This has been reverted in favor of changes in the model.